tests/README.md

```diff
@@ -1,32 +1,40 @@
 # Rook Test Framework
 
-The Rook Test Framework is a Go project that that uses a Docker container with Kubernetes
-tools and Docker pre-installed to quickly create a three node Kubernetes cluster using Docker in
-Docker (DinD) techniques, installs Rook and then runs automation tests to verify its functionality.
+The Rook Test Framework is used to run end to end and integration tests on rook. The framework depends kubernetes
+that is running. The framework also provides scripts for starting kubernetes using kubeadm or minikube, so users can 
+quickly spin a kubernetes platform. The Kubeadm-dind.sh script starts a 3 node kubernetes(using docker in docker technique) 
+and the minikube.sh starts a single node kubernetes. The Test framework is designed to install rook, run tests and uninstall rook
 
 ## Requirements
 
 1. Docker version => 1.2 && < 17.0
 2. Ubuntu Host 16 (currently the framework has only been tested on this version)
+3. Kubernetes with kubectl configured
+4. Rook
 
 ## Instructions
 
-On your Ubuntu box do the following;
-1.   Be sure to allocate a minimum of 4 processors to your vm
-2.   Navigate to the root directory of the repo and run the following:
-         ```e2e/scripts/smoke_test.sh```
+###Set up
+#### Install Kubernetes
+Users can choose any kubernetes flavor of their choice.  The Test framework only depends on kubectl being configured. 
+The framework also provides scripts to install Kubernetes. Run [kubeadm-dind.sh](/tests/scripts/kubeadm-dind.sh) to setup
+a three node kubernetes cluster using kubeadm(using docker in docker. each node is a docker container running kubernetes) 
+or run [minikube.sh](/tests/scripts/minikube.sh)to set up a single node kubernetes using kubeadm. 
+  
+  Both the start up scrits are designed to start kubrnetes, make sure rbd is working and copy latest rook and toolbox images
+  to the kubernetes containers.
 
```

Comment:
you should note that minikube does no currently include cephfs.ko. @kokhang is going add it